# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/WebGLParent.h
# Commit: 204b899f436d
# Full Hash: 204b899f436d99f0eae759af3ccfcbaabfbbb428
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-11 09:34:54
# Regressor Bug: 1632249
# File Overlap Count: 3
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r 3f5baab34b1f -r 204b899f436d dom/canvas/WebGLParent.h
--- a/dom/canvas/WebGLParent.h	Thu Jun 11 06:36:55 2020 +0000
+++ b/dom/canvas/WebGLParent.h	Thu Jun 11 06:37:35 2020 +0000
@@ -37,13 +37,10 @@
       const webgl::InitContextDesc&, UniquePtr<HostWebGLCommandSinkP>&& aSinkP,
       UniquePtr<HostWebGLCommandSinkI>&& aSinkI, webgl::InitContextResult* out);
 
-  RefPtr<layers::SharedSurfaceTextureClient> GetVRFrame(webgl::ObjectId);
-
   // Drain the command queue now.  Used by synchronous IpdlQueue consumers.
   bool RunQueue(uint64_t) { return RunCommandQueue(); }
 
-  // For IPDL:
-  WebGLParent();
+  WebGLParent();  // For IPDL
 
  private:
   ~WebGLParent();